### PR TITLE
v0.16 Docs

### PR DESCRIPTION
**Pull request by ryanahamilton**

### CODE REVIEW

**Comment by paolaperaza** [comment]
@vishwas-astro Haven't 100% polished these, but feel free to take a peek at the 0.16 release notes and let me know if I'm missing anything notable.

File: v0.16/release-notes.md [hunk 1]
```diff
@@ -5,6 +5,42 @@ date: 2020-01-24T00:00:00.000Z
...
+- Security: Implement platform-wide Pod Security Policies
+- Security: Clear Javascript CVEs
+
+## Astronomer Enterprise-only Changes
+
+### Support for AD FS
+
+Astronomer v0.16 for Astronomer Enterprise users comes with support for Microsoft's [Active Directory Federation services (AD FS)](https://docs.microsoft.com/en-us/windows-server/identity/active-directory-federation-services), as an alternative authentication system.
+
+To learn more, reference ["Auth Systems on Astronomer"](https://www.astronomer.io/docs/ee-integrating-auth-systems/).12
```
Comment (vishwas-astro) [inline v0.16/release-notes.md:36]: what is the .12 at the end of this line?

**Comment by vishwas-astro** [comment]
@paolaperaza looks good. just left a minor comment on the 0.16 release notes section.

File: v0.16/release-notes.md (hunk 1 above)
Comment (paolaperaza) [reply v0.16/release-notes.md:36]: Whoops - typo. Adjusting!

**Comment by paolaperaza** [comment]
Alright, added the new Env Var doc and I believe we are all set! Approving this PR but @petedejoy feel free to take a look as well.

@ryanahamilton Will you be able to take care of website changes needed to add 0.16 to the versioning menu? 🙏

**Comment by ryanahamilton** [comment]
@paolaperaza yeah, I already have the website update ready: astronomer/website#687

**Comment by paolaperaza** [comment]
@ryanahamilton You are :hero:. In that case, feel free to merge! My Env Var changes doc looks good on preview and is in the menu: https://preview.astronomer.io/docs/environment-variables/

File: v0.16/environment-variables.md [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+
+![Astro UI Env Vars Config](https://assets2.astronomer.io/main/docs/astronomer-ui/v0.16-Astro-UI-EnvVars.png)
+
+> **Note:** Input for all configurations officially supported by Airflow are pre-templated, but you're free to specify your own values.
+
+#### Mark Environment Variables as "Secret"
+
+On Astronomer, users have the ability to mark any Environment Variable as "secret" via the UI. For those who have Environment Variables containing potentially sensitive information (e.g. SMTP password, S3 bucket, etc.), we'd recommend leveraging this feature.
+
+To do so from the `Variables` tab,
```
Comment (ryanahamilton) [inline v0.16/environment-variables.md:144]: @paolaperaza I'm not sure why I haven't really noticed this before, but I see that this pattern is pretty widespread throughout our docs. The semantics don't seem appropriate for us to indicate verbatim labels as `code`—they'd probably be more appropriate within quotations and/or bolded.

```suggestion
To do so from the "**Variables**" tab,
```

This section being a good example:

| Before | After |
|---|---|
| image  | image  |

Thoughts? I'd love to start capturing writing guidelines such as these in out new docs repo.

Comment (paolaperaza) [reply v0.16/environment-variables.md:144]: @ryanahamilton Didn't see this comment earlier, but great callout and I 100% agree. TBH I think I'm mostly the one to blame for having those labels/buttons be formatted in "code" - it didn't quite feel right but I never really considered an alternative. I'm a fan of the formatting you suggested - I'll work to fix those as I see them and we can plan on a deeper clean once we've changed repos.

Will be great once we're working off of a new repo to generate writing guidelines for anyone contributing to docs 😊

**Pull request merged**